demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>Out of curiosity, how long do these cells take at the moment?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>This particular line took for me</p><pre class="ql-syntax" spellcheck="false">CPU times: user 6min 12s, sys: 2.82 s, total: 6min 15s
Wall time: 5min 43s
</pre>